lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/DefaultRemoteManagerFactory.java

```diff
@@ -69,7 +66,7 @@ private DefaultRemoteManagerFactory(
   @Override
   public RemoteManager getInstance(final String name) {
     try {
-      final Injector newInjector = injector.forkInjector();
+      final Injector newInjector = Tang.Factory.getTang().newInjector();
```

Comment:
All of these injectors are the same and get the same `bindVolatile()` calls. That opens two questions:

  1. Can we move the creation of the Injector into a `private` utility method? Once we fix this for good, that would limit the number of lines we have to touch then.
  2. Can we make the Injector an instance variable?